index.html

```diff
@@ -0,0 +1,173 @@
+            <div class="program-list">
+                <!-- Card 1 -->
+                <div class="program-card">
+                    <img src="imgs/distribution-icon.png" alt="Marketplace">
+                    <h3 class="program-title">Distribution</h3>
+                    <p>“The best supply chain is one that has no beginning and no end.”</p>
+                </div>
+                <!-- Card 2 -->
+                <div class="program-card">
+                    <img src="imgs/digital-marketing.png" alt="Affiliate">
+                    <h3 class="program-title">Marketing</h3>
+                    <p>“Business has only two functions — marketing and innovation.”</p>
+                </div>
+                <!-- Card 3 -->
+                <div class="program-card">
+                    <img src="imgs/promotion.png" alt="blog">
+                    <h3 class="program-title">Promotion</h3>
+                    <p>“It's not what you sell that matters as much as how you sell it!”</p>
+                </div>
+                <!-- Card 4 -->
+                <div class="program-card">
+                    <img src="imgs/educate.png" alt="Education">
+                    <h3 class="program-title">Education</h3>
+                    <p>“Education is one thing no one can take away from you.”</p>
+                </div>
+            </div>
+            <a href="#" class="program-text" type="button"> SEE THE WHOLE PROGRAM</a>
+        </section>
+
+        <!-- <section class="blog">
+            <div class="rightimg">
+                <div class="blog1">
+                    <img src="imgs/video-conference-meeting.jpg" alt="">
+                </div>
+                <div class="blog1">
+                    <img src="imgs/crowd-participating-at-event.jpg" alt="">
+                </div>
+            </div>
+            <div class="leftimg">
+                <div class="blog1">
+                    <img src="imgs/video-conference-meeting.jpg" alt="">
+                </div>
+                <div class="blog1">
+                    <img src="imgs/crowd-participating-at-event.jpg" alt="">
+                </div>
+            </div>
+        </section> -->
+        <section class="speakers-container" id="news">
+            <h2>Featured speakers</h2>
+            <div class="speakers"></div>
+            
+        </section>
+
+        <section class="sponsor" id="sponsor">
+            <h2>Partners</h2>
+            <ul class="sponsor-brand">
+                <li class="sponsor-list">
+                    <img src="imgs/microverselogo.webp" alt="sponsor1" class="spon1">
+                </li>
+
+                <li class="sponsor-list">
+                    <img src="imgs/cisco-logo-transparent.png" alt="sponsor2" class="spon1">
+                </li>
+
+                <li class="sponsor-list">
+                    <img src="imgs/FreeCodeCamp_logo.png" alt="sponsor2" class="spon1">
+                </li>
+            </ul>
+        </section>
```

Comment:
Please kindly include the second section of the footer

### Your design
![image](https://user-images.githubusercontent.com/78251927/212152050-e5bc3892-50ab-40c2-ad65-75a0b0a0700d.png)

### Template
![image](https://user-images.githubusercontent.com/78251927/212151805-669fe2c3-0027-4f9f-a99b-e68a4bfda50d.png)